backend/src/main/java/com/htecgroup/skynest/model/entity/BucketEntity.java

```diff
@@ -28,4 +27,10 @@ public class BucketEntity extends ObjectEntity {
   @Column(name = "public")
   @With
   private Boolean isPublic;
+
+  @ElementCollection(targetClass = LambdaType.class)
+  @CollectionTable(name = "lambda_on_bucket", joinColumns = @JoinColumn(name = "bucket_id"))
+  @Enumerated(EnumType.STRING)
+  @Column(name = "lambda_name")
+  private List<LambdaType> lambdaTypes;
```

Comment:
Can it be simpler, for example oneToMany in annotations like in some other entities?

Reply:
LambdaType is an enum, so I can't do that. If I make a class I'm not sure how the factory is going to create lambdas. I'm looking now at some issues like if I change the name of some value in the enum it will break the database. I'm gonna look that up and see if I can find a solution.